api/python/cellxgene_census/src/cellxgene_census/experimental/util/_csr_iter.py

```diff
@@ -81,42 +100,51 @@ def X_sparse_iter(
     obs_coords = query.obs_joinids().to_numpy()
     obs_coord_chunker = (obs_coords[i : i + stride] for i in range(0, len(obs_coords), stride))
 
-    # Lazy read into Arrow Table. Yields (coords, Arrow.Table)
     X = query._ms.X[X_name]
     var_coords = query.var_joinids().to_numpy()
+
+    # Lazy read of X chunk. Yields ((obs_coords, var_coords), (soma_data, soma_dim_0, soma_dim_1))
```

Comment:
comment says this is lazy, but it is not.  It always returns a materialized tuple.

I suspect this is an error with the comment, as the `table_reader` is still lazy, and there is no benefit to making the table-to-tuple-of-numpy conversion lazy.  

Suggest just fixing the comment - perhaps by moving it down a few lines